node_modules/col-activities/lib/notifications/daily.js

```diff
@@ -493,13 +494,35 @@ var getCourseAssetData = function(ctx, users, dateRange, callback) {
     'include': [
       // Ideally, we'd just get the IDs and look up the objects in the `users` array ourselves
       // to reduce data serialization. Unfortunately, Sequelize doesn't like this very much,
-      // so we simple get the basic user profiles
-      {'model': DB.User, 'as': 'users', 'required': true, 'attributes': UserConstants.BASIC_USER_FIELDS},
-
+      // so we simply get the basic user profiles
+      {
+        'model': DB.User,
+        'as': 'users',
+        'required': true,
+        'attributes': UserConstants.BASIC_USER_FIELDS
+      },
+      // Get category visibility information to avoid surfacing hidden assets
+      {
+        'model': DB.Category,
+        'attributes': ['visible']
+      },
       // The comments
-      {'model': DB.Comment, 'required': true, 'where': {'created_at': {'gt': dateRange.start, 'lt': dateRange.end}}, 'include': [
-        {'model': DB.Comment, 'required': false, 'as': 'parent'}
-      ]}
+      {
+        'model': DB.Comment,
+        'required': true,
+        'where':
+        {
+          'created_at': {
```

Comment:
@paulkerschen minor concern: do we have a varying code style above w.r.t opening curly brace? Do we prefer:
```
  'created_at':
  {
    ...
```